connect_ext_test.go

```diff
@@ -387,6 +388,44 @@ func TestServer(t *testing.T) {
 	})
 }
 
+func TestConcurrentStreams(t *testing.T) {
+	if testing.Short() {
+		t.Skipf("skipping %s test in short mode", t.Name())
+	}
+	t.Parallel()
+	mux := http.NewServeMux()
+	mux.Handle(pingv1connect.NewPingServiceHandler(pingServer{}))
+	server := httptest.NewUnstartedServer(mux)
+	server.EnableHTTP2 = true
+	server.StartTLS()
+	t.Cleanup(server.Close)
+	var done sync.WaitGroup
+	var startWg sync.WaitGroup
+	startWg.Add(1)
```

Comment:
Huh, I'm in the habit of using a plain `chan struct{}` instead of start wait group. I like this more!

What do you think of making this a one-liner? `var start, done sync.WaitGroup`